Dockerfile

```diff
@@ -2,11 +2,12 @@
 FROM python:3.9
 
 # Set working directory.
-
+WORKDIR /arpanGupta--jtu-22-assignment
 # Expose port 8000 for your FastAPI server.
-
+EXPOSE 8000
 # Install Python requirements using Pip.
-
+RUN pip install --no-cashe-dir -r requirements.txt
```

Comment:
You need to copy `requirements.txt` to your Docker container before being able to install any requirements from it. At this point in your Docker container, it doesn't contain the `requirements.txt` file.

Reply:
Understood. I have updated the file, please check.